elasticgraph-graphql/spec/unit/elastic_graph/graphql/datastore_query/search_index_expression_spec.rb

```diff
@@ -369,6 +369,24 @@ class GraphQL
 
               expect(parts).to target_all_widget_indices
```

Comment:
Here, the following returns all indices, which is different then how we treat anyOf when filtering a query. Is this okay, or should we make changes to make sure everything is more aligned? 

```
              parts = search_index_expression_parts_for({"any_of" => []})
```